File: AspNetCore/FollowMePlain/src/FollowMePlain/Data/BlogItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using MongoDB.Bson;

namespace FollowMePlain.Data
{
    public class BlogItem
    {
        public ObjectId Id { get; set; }
        public string Url { get; set; }
        public string Name { get; set; }
        public string RssFeedLink { get; set; }
        public DateTime? FollowDate { get; set; }
        public List<BlogItemPost> BlogItemPosts { get; set; }
    }
}

File: NodeJs/FollowMeNode/app/models/blogItem.js
var mongoose = require("mongoose");

var Schema = mongoose.Schema;

var BlogItemSchema = new Schema({

    Url: { type: String },
    Name: { type: String },
    FollowDate: { type: Date, default: Date.now },
    RssFeedLink: { type: String },
    BlogItemPosts: { type: Array }

}, { collection: "BlogItems" });

module.exports = mongoose.model("BlogItem", BlogItemSchema);